app/middleware/parameter-validation.ts

```diff
@@ -40,6 +59,9 @@ export default function parameterValidation(
 ): void {
   try {
     validateLinkTypeParameter(req);
+    if (req.url.match(RANGESET_ROUTE_REGEX)) {
```

Comment:
I tested the WMS route and see it allows bogus parameters still. Should we continue to ignore extra parameters there?

http://localhost:3000/C1233800302-EEDTEST/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&BBOX=-180,-90,154.4170924291685765,90&CRS=CRS%3A84&WIDTH=2000&HEIGHT=1078&LAYERS=C1233800302-EEDTEST&STYLES&FORMAT=image%2Fpng&DPI=72&MAP_RESOLUTION=72&FORMAT_OPTIONS=dpi%3A72&TRANSPARENT=TRUE&granuleId=G1233800343-EEDTEST&turbo=true&foo=bar

Reply:
I'd prefer we reject them, but I'd be okay with this being another ticket if it's more than 1-2 points of effort.